main.go

```diff
@@ -166,11 +166,16 @@ func main() {
 					commands.NewRemindersCommand(store, w, generateID, userParser),
 				}
 
-				args, err := utils.ParseShell(e.Msg.Text)
+				parseOutput, err := utils.ParseShell(e.Msg.Text)
```

Comment:
```
args, err := utils.Parse(...)
...

args = append([]string{""}, args...)
```

Reply:
Ah, I forgot the variadric portion of append when I first attempted this.